src/main/java/io/ebeaninternal/server/query/SqlBeanLoad.java

```diff
@@ -88,8 +86,9 @@ public Object load(BeanProperty prop) {
       return dbVal;
 
     } catch (Exception e) {
```

Comment:
We should probably only setLoadError on TextException ... and let everything else fail by throwing PersistenceException